datadog/tests/cassettes/TestAccDatadogSyntheticsPrivateLocation_Basic.yaml

```diff
@@ -6,14 +6,14 @@ interactions:
         proto: HTTP/1.1
         proto_major: 1
         proto_minor: 1
-        content_length: 137
+        content_length: 151
         transfer_encoding: []
         trailer: {}
         host: api.datadoghq.com
         remote_addr: ""
         request_uri: ""
         body: |
-            {"description":"a private location","name":"tf-TestAccDatadogSyntheticsPrivateLocation_Basic-local-1714765065","tags":["foo:bar","baz"]}
+            {"description":"a private location","metadata":{},"name":"tf-TestAccDatadogSyntheticsPrivateLocation_Basic-local-1741182374","tags":["foo:bar","baz"]}
```

Comment:
Since there's a bug fix here, I suggest you either:
* Split the PR so the bug fix is done in isolation (better)
* Mark this PR as a bugfix (slightly worse because it makes it harder to read what the bug fix exactly is)